ci/deploy_credentials.yaml

```
@@ -3,11 +3,11 @@ image_resource:
   type: docker-image
   source:
     repository: gcr.io/google.com/cloudsdktool/cloud-sdk
+    tag: latest
```

For `gcloud secrets` the alpine image is sufficient.
```suggestion
    tag: alpine
```